draft-packages/button/KaizenDraft/Button/decision-tokens.scss

```diff
@@ -0,0 +1,24 @@
+@import "~@kaizen/design-tokens/sass/color";
+
+// Zen theme
+$dt-color-background-active: $kz-color-wisteria-100;
+$dt-color-background-hover: $kz-color-wisteria-100;
+$dt-color-background-focus: $kz-color-wisteria-100;
+$dt-color-border-color-base: $kz-color-wisteria-500;
+$dt-color-border-color-hover: $kz-color-wisteria-700;
+$dt-color-border-color-focus: $kz-color-wisteria-700;
+$dt-color-border-color-disabled: $kz-color-wisteria-500;
+$dt-color-color-disabled: $kz-color-wisteria-800;
```

Comment:
I don't think this whole section is needed, the fallback values below are all that will be used. (I _think_. SASS is weird so I could be wrong).

Reply:
Fixed